components/header/Header.tsx

```diff
@@ -0,0 +1,81 @@
+'use client';
+import React from 'react';
+import {
+  Avatar,
+  ButtonMenu,
+  EAvatarSizes,
+  EIconColors,
+  EIConTypes,
+  ELogoColors,
+  ELogoPositions,
+  Icon,
+  Logo,
+} from '@ost-cas-fee-adv-23-24/elbmum-design';
+import Link from 'next/link';
+import { signOut, signIn } from 'next-auth/react';
+import useProfileInfo from '@/hooks/useProfileInfo';
+
+interface IProps {}
+
+const Header = ({}: IProps) => {
```

Comment:
Perhaps you could rename it to "Navigation" as in Figma Design. Moreover, my profile header component is also called "Header" at the moment.

Reply:
Done